cpt/packager.py

```diff
@@ -196,7 +198,7 @@ def __init__(self, username=None, channel=None, runner=None,
         self.sudo_pip_command = ""
         if "CONAN_PIP_USE_SUDO" in os.environ:
             self.sudo_pip_command = "sudo -E" if get_bool_from_env("CONAN_PIP_USE_SUDO") else ""
-        elif platform.system() != "Windows":
+        elif platform.system() != "Windows" and 'conanio/' not in str(self._docker_image):
```

Comment:
If not specified I try to use our images too, so probably if no docker image is specified we have to skip
` elif platform.system() != "Windows" and (self._docker_image and 'conanio/' not in self._docker_image):`

Reply:
`elif platform.system() != "Windows" and not self.use_docker`

?

Reply:
Mhhh, I don't think so, if not self.use_docker then it uses our images, so no sudo is required. right?

Reply:
oh, wait

Reply:
Mhh I don't get it, I want to use docker, so I specified CONAN_USE_DOCKER but I haven't specified CONAN_DOCKER_IMAGE, so it will use our images. So no sudo is adjusted.

Reply:
done!